fortio_main.go

```diff
@@ -114,12 +114,15 @@ var (
 	syncFlag    = flag.String("sync", "", "index.tsv or s3/gcs bucket xml URL to fetch at startup for server modes.")
 	baseURLFlag = flag.String("base-url", "",
 		"base URL used as prefix for data/index.tsv generation. (when empty, the url from the first request is used)")
+	newMaxPayloadSizeKb int
 )
 
 func main() {
 	flag.Var(&headersFlags, "H", "Additional Header(s)")
 	flag.IntVar(&fhttp.BufferSizeKb, "httpbufferkb", fhttp.BufferSizeKb,
 		"Size of the buffer (max data size) for the optimized http client in kbytes")
+	flag.IntVar(&newMaxPayloadSizeKb, "maxpayloadsizekb", fhttp.MaxPayloadSize/1024,
```

Comment:
just put it with the other flags, as a flag.Int() doesn't need to use a variable

Reply:
fixed.